modules/userId/index.js

```diff
@@ -315,6 +315,60 @@ function hasGDPRConsent(consentData) {
   return true;
 }
 
+/**
+   * Find the root domain
+   * @param {string|undefined} fullDomain
+   * @return {string}
+   */
+function findRootDomain(fullDomain = document.domain) {
```

Comment:
`document.domain` is deprecated and according to [this article](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain) some modern browsers may not support it.

Reply:
updated